src/js/tab.js

```diff
@@ -95,7 +95,7 @@ Tab.prototype.createAside = function(content) {
  * @param content
  */
 Tab.prototype.createFooter = function(content) {
-  var footer = $('<footer>' + content + '</footer>');
+  var footer = content ? $('<footer>' + content + '</footer>') : $('<footer></footer>');
```

Comment:
What made this change necessary?
However, I think the ternary operator can be avoided here by setting `content` to an empty string if it is falsy (null, false, 0, ...).
`if (!content) { content = '' }`
As this construct raises a warning with our current eslint settings (to check run `gulp lint`).